README.md

```diff
@@ -27,6 +27,7 @@ salita
  - `--ignore-stars`: ignore updates to packages that are set to "*"
  - `--ignore-pegged`: ignore updates to packages that are pegged to a single version, rather than a range
  - `--check`: implies "dry-run"; and returns with an exit code matching the number of updated dependencies.
+ - `--sections`: comma-separated list of sections to process, default: dep,dev,pee,opt.
```

Comment:
`s/pee/peer/g`, there's no need to make them all be 3 letters - and "pee" isn't really a good word choice :-)